falcon/response.py

```diff
@@ -41,10 +41,6 @@
 from falcon.util.uri import encode_check_escaped as uri_encode
 from falcon.util.uri import encode_value_check_escaped as uri_encode_value
 
-_STREAM_LEN_REMOVED_MSG = (
```

Comment:
It seems that `ruff` is complaining about the import order, probably just a matter of missing or extraneous newline after imports. Could you simply try `ruff check --fix`, and see if it solves the problem?